docs/advanced-features/react-18/server-components.md

```diff
@@ -32,7 +32,7 @@ To run a component on the server, append `.server.js` to the end of the filename
 
 For client components, append `.client.js` to the filename. For example, `./components/avatar.client.js`.
 
-You can then import other server or client components from any server component. Note: a server component **can not** be imported by a client component. Components without "server/client" extensions will be treated as shared components and can be used and rendered by both sides, depending on where it is imported. For example:
+You can then import other server or client components in any server component. Note: a server component **can not** be imported by a client component. Components without "server/client" extensions will be treated as shared components and can be used and rendered by both sides, depending on where it is imported. For example:
```

Comment:
I’m thinking we should flip these sentences around. Something like “A server component can import both server components and client components. However, client components cannot import server components.